src/community/security/oauth2-openid-connect/src/main/java/org/geoserver/security/oauth2/ClientSecretRequestEnhancer.java

```diff
@@ -0,0 +1,19 @@
+package org.geoserver.security.oauth2;
```

Comment:
Missing copyright header

Reply:
thanks for the point, I've added  it together with a call description.